Index: build/build.sh
===================================================================
--- build/build.sh	(revision 1033)
+++ build/build.sh	(working copy)
@@ -493,7 +493,9 @@
      BIN_PATH=$BIN_PATH \
      OBJ_PATH=$OBJ_PATH/release \
      release
-     
+
+exit $?
+
 if test $? -gt 0; then
    exit $?
 fi
@@ -542,4 +544,4 @@
         exit $?
     fi
 fi   
- 
\ No newline at end of file
+ 
